cypress/integration/page.spec.js

```diff
@@ -396,7 +425,7 @@
       });
 
       it('should show posts loader while waiting for API response', () => {
-        page.mockUser1Posts()
+        page.mockUser1Posts();
```

Comment:
There is a missing semicolon at the end of the `cy.intercept` statement. Ensure that each statement is properly terminated to avoid syntax errors.